🧑‍🌾 gz_sensors-ci-gz-sensors10-homebrew-amd64 clang warning related to `DopplerVelocityLog.cc`

<!-- If you're not sure whether your problem is a bug, please ask a question at
https://robotics.stackexchange.com instead.-->

## Environment
* OS Version: mac sonoma
* Source or binary build?
  source

## Description
* Expected behavior: The test to pass without warnings
* Actual behavior: Test is dropping a clang warning `jacobiSvd` is deprecated

## Steps to reproduce
<!-- Provide steps so we can try to reproduce this issue -->

1. Go to https://build.osrfoundation.org/job/gz_sensors-ci-gz-sensors10-homebrew-amd64
2. Build the job
3. See the warning appear

## Output
```
/Users/jenkins/jenkins-agent/workspace/gz_sensors-ci-gz-sensors10-homebrew-amd64/gz-sensors/src/DopplerVelocityLog.cc:1575:47: warning: 'jacobiSvd' is deprecated: Options should be specified using method's template parameter. [-Wdeprecated-declarations]
 1575 |             beamBasis.topRows(numBeamsLocked).jacobiSvd(
      |                                               ^
/usr/local/include/eigen3/Eigen/src/Core/MatrixBase.h:377:42: note: 'jacobiSvd' has been explicitly marked deprecated here
  377 |   inline JacobiSVD<PlainObject, Options> jacobiSvd(unsigned int computationOptions) const;
      |                                          ^
/Users/jenkins/jenkins-agent/workspace/gz_sensors-ci-gz-sensors10-homebrew-amd64/gz-sensors/src/DopplerVelocityLog.cc:1575:47: warning: 'jacobiSvd<0>' is deprecated: Options should be specified using method's template parameter. [-Wdeprecated-declarations]
 1575 |             beamBasis.topRows(numBeamsLocked).jacobiSvd(
      |                                               ^
/usr/local/include/eigen3/Eigen/src/Core/MatrixBase.h:376:3: note: 'jacobiSvd<0>' has been explicitly marked deprecated here
  376 |   EIGEN_DEPRECATED_WITH_REASON("Options should be specified using method's template parameter.")
      |   ^
/usr/local/include/eigen3/Eigen/src/Core/util/Macros.h:965:62: note: expanded from macro 'EIGEN_DEPRECATED_WITH_REASON'
  965 | #define EIGEN_DEPRECATED_WITH_REASON(message) __attribute__((deprecated(message)))
      |                                                              ^
/Users/jenkins/jenkins-agent/workspace/gz_sensors-ci-gz-sensors10-homebrew-amd64/gz-sensors/src/DopplerVelocityLog.cc:1825:47: warning: 'jacobiSvd' is deprecated: Options should be specified using method's template parameter. [-Wdeprecated-declarations]
 1825 |             beamBasis.topRows(numBeamsLocked).jacobiSvd(
      |                                               ^
/usr/local/include/eigen3/Eigen/src/Core/MatrixBase.h:377:42: note: 'jacobiSvd' has been explicitly marked deprecated here
  377 |   inline JacobiSVD<PlainObject, Options> jacobiSvd(unsigned int computationOptions) const;
      |                                          ^
/Users/jenkins/jenkins-agent/workspace/gz_sensors-ci-gz-sensors10-homebrew-amd64/gz-sensors/src/DopplerVelocityLog.cc:1825:47: warning: 'jacobiSvd<0>' is deprecated: Options should be specified using method's template parameter. [-Wdeprecated-declarations]
 1825 |             beamBasis.topRows(numBeamsLocked).jacobiSvd(
      |                                               ^
/usr/local/include/eigen3/Eigen/src/Core/MatrixBase.h:376:3: note: 'jacobiSvd<0>' has been explicitly marked deprecated here
  376 |   EIGEN_DEPRECATED_WITH_REASON("Options should be specified using method's template parameter.")
      |   ^
/usr/local/include/eigen3/Eigen/src/Core/util/Macros.h:965:62: note: expanded from macro 'EIGEN_DEPRECATED_WITH_REASON'
  965 | #define EIGEN_DEPRECATED_WITH_REASON(message) __attribute__((deprecated(message)))
```

CC: @Crola1702

--- a/src/DopplerVelocityLog.cc
+++ b/src/DopplerVelocityLog.cc
@@ -1573,9 +1573,9 @@ namespace gz
         // Enough rows for a unique least squares solution
         const Eigen::MatrixXd svdMat =
           beamBasis.topRows(numBeamsLocked).eval();
-        using SvdT = Eigen::JacobiSVD<Eigen::MatrixXd,
-            Eigen::ComputeThinU | Eigen::ComputeThinV>;
-        const SvdT svdDecomposition(svdMat);
+        using SvdT = Eigen::JacobiSVD<Eigen::MatrixXd>;
+        const SvdT svdDecomposition(svdMat,
+          Eigen::ComputeThinU | Eigen::ComputeThinV);
 
         // Estimate DVL velocity mean and covariance in the reference frame
         const Eigen::Vector3d velocityMeanInReferenceFrame =
@@ -1825,9 +1825,9 @@ namespace gz
         // Enough rows for a unique least squares solution
         const Eigen::MatrixXd svdMat =
           beamBasis.topRows(numBeamsLocked).eval();
-        using SvdT = Eigen::JacobiSVD<Eigen::MatrixXd,
-            Eigen::ComputeThinU | Eigen::ComputeThinV>;
-        const SvdT svdDecomposition(svdMat);
+        using SvdT = Eigen::JacobiSVD<Eigen::MatrixXd>;
+        const SvdT svdDecomposition(svdMat,
+          Eigen::ComputeThinU | Eigen::ComputeThinV);
 
         // Estimate DVL velocity mean and covariance in the reference frame
         const Eigen::Vector3d velocityMeanInReferenceFrame =

